tests/jest-dom-matchers/toBeValid.test.ts

```diff
@@ -0,0 +1,38 @@
+import { createTab } from 'test-mule';
+
+test('toBeValid', async () => {
```

Comment:
@emersonthis What do you think about combining this test with `toBeValid` since it is very similar?